modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js

```diff
@@ -142,7 +150,51 @@
 		},
 
 		/**
-		 * Checks if the pasted data is image and passes it to
+		 * Handler for when images are uploaded
+		 *
+		 * @instance
+		 * @memberof CKEDITOR.plugins.addimages
+		 * @method _onImageUploaded
+		 * @param {Object} event Event data
```

Comment:
I see that `_onPaste` method is receiving
```
* @param {CKEDITOR.dom.event} event A `paste` event, as received natively from CKEditor
```
Are we receiving the same type of param here?

Reply:
@carloslancha that's actually a good point:

If you look at the [docs](https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_editor.html#event-paste) for the `paste` event it actually says that the first argument (usually called `evt` in the CKEDITOR "ecosystem"), is of type `eventInfo`. [Here](https://github.com/ckeditor/ckeditor4/blob/eb2d11644a796cd13cd047cd55541430359f9317/core/eventInfo.js#L12-L16)'s the definition of `eventInfo` that I see in CKEDITOR's source code - note that there is a comment saying 
>  * This class is not really part of the API.


For events that we dispatch ourselves, for example, when calling the `fire` method, like it's the case here, we'll also receive an `eventInfo` object. 

While it might make sense to update this, I personally don't know what value these JSDoc comments bring, but that's just a personal preference.

You probably want to reply "if this was TypeScript it wouldn't happen" and you'd be right.
I'll update this, but I personally wouldn't add JSDoc comments to CKEDITOR plugins.

Reply:
> I'll update this, but I personally wouldn't add JSDoc comments to CKEDITOR plugins.

Hot-take: JSDoc is a little overrated... 😂

![overrated](https://user-images.githubusercontent.com/7074/89875204-41f37e80-dbbd-11ea-9167-33bd2eeec005.gif)

Reply:
> Me? TypeScript??

:rofl: 

> I'm fine with that, just pointed it out to keep consistency. If we don't want to use JSDocs... let's get rid of them! :D

Yes and that was a good idea, I updated them so I think we can keep it for now